web/src/components/ToggleButton.tsx

```diff
@@ -53,14 +51,14 @@ export default function ToggleButton({
             key={`group-item-${key}`}
             value={option.value}
             onClick={() => onToggle(option.value)}
-            className={`
-       inline-flex h-[26px] w-full  items-center whitespace-nowrap rounded-full px-4 ${fontSize} ${
+            className={twMerge(
+              'inline-flex h-[26px] w-full items-center whitespace-nowrap rounded-full px-4 text-xs',
               option.value === selectedOption
-                ? ' bg-white  shadow transition duration-500 ease-in-out dark:bg-gray-500'
+                ? ' bg-white text-brand-green  shadow transition duration-500 ease-in-out dark:bg-gray-500'
                 : ''
-            }`}
+            )}
           >
-            <p className="sans flex-grow select-none  dark:text-white">
+            <p className="sans flex-grow select-none font-inter font-bold capitalize  dark:text-white">
```

Comment:
If we make this bold we should we make these bold as well?

![image](https://user-images.githubusercontent.com/30777521/224678781-a60a3e74-3be8-48dc-96d6-5d1c60c86143.png)

Reply:
I think so! @rasmusjeppsen what do you think?

Reply:
Something to add here as well: Don't we use inter for basically everything (with a few exceptions)? Would it not be easier to set the font style on the root of the html tag or the body tag and override it where needed instead?

Reply:
Yeah, it's already our default so I didn't need to specify here at all 🤦